src/components/seeds/summary/SummaryPaper.tsx

```diff
@@ -10,12 +10,13 @@ interface Props {
   statistic?: number | string;
   loading: boolean;
   error: boolean;
+  tooltipTitle?: NonNullable<React.ReactNode>;
```

Comment:
Why don't we use a string here?

Reply:
As mentioned in [the related PR](https://github.com/terraware/web-components/pull/76), the type matches the type for the `title` prop on the `Tooltip` component. We need to be able to pass in some HTML and/or other React elements for some of the tooltips.